openupgradelib/openupgrade.py

```diff
@@ -1778,29 +1760,35 @@ def wrapped_function(cr, version):
                     # Python3: fetch pyfile from locals, not fp
                     filename = frame.locals.get(
                         'pyfile') or frame.locals['fp'].name
-                except Exception as e:
+                except Exception as exc:
```

Comment:
Please don't rename this variable for not increasing diff

Reply:
pylint is loadly complaining about one-letter variable-name, as well as about using `%` formatter to create logging messages.

Reply:
I try not to create to big a diff, therefor only changed the functions that had to change anyway.

Reply:
But that complaint is only a warning AFAIK.